hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/handlers/HandlerUtility.java

```diff
@@ -111,6 +108,50 @@ static TransactionBody asOrdinary(@NonNull final Schedule scheduleInState) {
         return ordinary.build();
     }
 
+    static HederaFunctionality functionalityForType(final DataOneOfType transactionType) {
```

Comment:
Is there such a function already in HapiUtils? If not, maybe this should move there?

Reply:
I couldn't find one when I wrote this 4 months back.  I'll do some searching and see what I find.